packages/terra-spacer/CHANGELOG.md

```diff
@@ -18,6 +22,7 @@ Unreleased
 ------------------
 ### Changed
 * Minor dependency version bump
+=======
```

Comment:
Whoops. Looks like this got missed in a merge

Reply:
Fixed 37e472c0